Robot2018/src/org/usfirst/frc/team199/Robot2018/Robot.java

```diff
@@ -131,6 +133,7 @@ public void disabledPeriodic() {
 	 */
 	@Override
 	public void autonomousInit() {
+		Scheduler.getInstance().add(new ShiftLowGear());
```

Comment:
Are we absolutely sure that this command won't get canceled (by another auto command that requires drivetrain) before it gets executed. It might be OK as is, but I'm not sure.

Reply:
This command doesn't require drivetrain, should be able to run even if another command that does require drivetrain runs.

Reply:
OK. My bad.